pkg/codegen/go/gen_program_expressions.go

```diff
@@ -681,7 +744,14 @@ func (g *generator) genApply(w io.Writer, expr *model.FunctionCallExpression) {
 		// If we only have a single output, just generate a normal `.Apply`
 		g.Fgenf(w, "%.v.ApplyT(%.v)%s", applyArgs[0], then, typeAssertion)
 	} else {
-		// TODO
+		g.Fgenf(w, "pulumi.All(%.v", applyArgs[0])
+		applyArgs = applyArgs[1:]
+		for _, a := range applyArgs {
+			g.Fgenf(w, ",%.v", a)
```

Comment:
What does the `%.v` format do differently than `%v`?

Reply:
I'm actually not sure the entirety of its behavior. I picked this up from the other code generators. 

It isn't explicitly documented in [pkg/fmt](https://golang.org/pkg/fmt/) as far as I can tell. It does cause truncation of floats similar to `%.f`, not sure if it has other behaviors:

https://play.golang.org/p/gr3O--qAIBM

Reply:
Perhaps @pgavlin can shed some light here. Looks like `%v` might translate itself to the default value (`%f` for floats) and interpret any format modifiers with that context.